it/src/test/scala/quasar/physical/s3/LWCTests.scala

```diff
@@ -1,302 +0,0 @@
-/*
- * Copyright 2014–2018 SlamData Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package quasar.physical.s3
-
-import slamdata.Predef._
-
-import org.specs2.execute.Result
-import org.specs2.mutable.Specification
-
-import pathy._
-import quasar.Data
-import quasar.contrib.pathy._
-import quasar.contrib.scalaz.concurrent.task._
-import quasar.fs.mount.ConnectionUri
-import quasar.mimir.LightweightFileSystem
-import scalaz._, Scalaz._
-import scalaz.concurrent.Task
-
-final case class TestFS(
-  // map from file paths to JSON data;
-  // if the file exists but is not parseable
-  // it's `None`
-  allFiles: Map[AFile, Option[List[Data]]],
-  // full directory structure
-  allPaths: Map[ADir, List[APath]]
-)
-
-final case class MultiCauseException(val msg: Option[String], val causes: List[Throwable]) extends Exception
-
-object LWCTests {
```

Comment:
Were all tests in here covered by `ResourceDiscoverySpec` or are there any that we should preserve? I saw there are some additional S3-specific tests in `S3DataSourceSpec` as well.